news_items/018_group_work/index.html

```diff
@@ -0,0 +1,61 @@
+<style>
+ img {
+     display: block;
+     margin-left: auto;
+     margin-right: auto;
+     margin-top: 15px;
+     margin-bottom: 15px;
+ }
+</style>
+
+<p>
+  Thanks to a huge development effort led
+  by <a href="https://github.com/frankyhao">Franky Hao</a>
+  and <a href="https://github.com/zhining4">Zhining Qiu</a>
+  PrairieLearn now supports group assessments. This lets students
+  collaboratively work on an assessment together, with everyone in the
+  group submitting answers to the same questions and everyone
+  receiving the same shared scores.
+</p>
+
+<img src="student_start_assessment.png" width="697px">
+
+<p>
+  There is a flexible group assignment system that allows instructors
+  to either assign students to groups (via CSV upload or randomly
+  generated by PrairieLearn) or let students make their own groups.
```

Comment:
And in the group config, instructors could set the maximum and minimum group size and students' permission (create/join/leave a group)

Reply:
Yep, I won't describe all the details in the news items (it links to the docs for full details). This news item is just to let people know that group work exists.